src/components/Card/Card.jsx

```diff
@@ -2,6 +2,7 @@ import { useState } from "react";
 import { FiMinus } from "react-icons/fi";
 import PropTypes from "prop-types";
 
+import { SiOpenai } from "react-icons/si";
```

Comment:
```suggestion
import PropTypes from "prop-types";
import { SiOpenai } from "react-icons/si";

```

- 라이브러리와 컴포넌트를 분리하면 좋을 것 같습니다!